src/OpenTelemetry/Metrics/MetricType.cs

```diff
@@ -23,10 +23,13 @@ public enum MetricType : byte
     {
         /*
         Type:
-            0x10: Sum
-            0x20: Gauge
-            0x30: Histogram
-            0x40: Summary (reserved)
+            0x10 0b00010000: Sum
+            0x20 0b00100000: Gauge
+            0x40 0b01000000: Histogram
+            0x50 0b01010000: HistogramWithBuckets
+            0x60 0b01100000: HistogramWithMinMax
+            0x70 0b01110000: HistogramWithMinMaxAndBuckets
```

Comment:
@reyang when exponentialhistogram arrives, we won't have any bits left for it.. unless we take Summary or make this 16 bits instead of current 8 bytes

Reply:
I think int is fine.